portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,7 +13,8 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import java.util.List;
```

Comment:
Can you order these alphabetically?  It will make it easier to know where to insert later.